modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -138,31 +89,45 @@ export default class ScenegraphLayer extends Layer {
   }
 
   updateState({props, oldProps, changeFlags}) {
-    if (changeFlags.propsChanged && props.gltf) {
-      const {scenes} = createGLTFObjects(this.context.gl, props.gltf, {
+    if (changeFlags.propsChanged) {
```

Comment:
Can skip this check

Reply:
Why?

Reply:
You are already manually comparing the two scenegraph props

Reply:
done